### DISCUSSION TITLE
Notion integration

### DISCUSSION THREAD

**Discussion by avivapinchas** [ParabolInc/parabol#5454] [New features]
Some teams (including our Growth team!) use Notion for task management. With Notion now having a public API, we could build an integration. 2 requests to date (including us). 

User requests: #3554
Issue converted to: #5660

**Comment by drewhousman1010** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/52486451) 🔒 writes:

> Our organization and multiple other small companies use Notion for documenting reports off of team meetings. It would be awesome to have this integration set up to use Parabol as part of our workflow

**Comment by drewhousman1010** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/15612601) 🔒 said:

> At the end of every retro we manually copy all the output into Notion. It would be really nice to have an integration.

**Comment by avivapinchas** [comment]
Notion's API is officially out of beta now. If we build an integration, we can ask to be included in their directory: https://notionup.typeform.com/to/kViev734?typeform-source=www.notion.so
